src/libs/ReportUtils.ts

```diff
@@ -2149,48 +2191,33 @@ function navigateToDetailsPage(report) {
  *
  * In a test of 500M reports (28 years of reports at our current max rate) we got 20-40 collisions meaning that
  * this is more than random enough for our needs.
- *
- * @returns {String}
  */
-function generateReportID() {
+function generateReportID(): string {
     return (Math.floor(Math.random() * 2 ** 21) * 2 ** 32 + Math.floor(Math.random() * 2 ** 32)).toString();
 }
 
-/**
- * @param {Object} report
- * @returns {Boolean}
- */
-function hasReportNameError(report) {
-    return !_.isEmpty(lodashGet(report, 'errorFields.reportName', {}));
+function hasReportNameError(report: OnyxEntry<Report>): boolean {
+    return Object.keys(report?.errorFields?.reportName ?? {}).length !== 0;
 }
 
 /**
  * For comments shorter than or equal to 10k chars, convert the comment from MD into HTML because that's how it is stored in the database
  * For longer comments, skip parsing, but still escape the text, and display plaintext for performance reasons. It takes over 40s to parse a 100k long string!!
- *
- * @param {String} text
- * @returns {String}
  */
-function getParsedComment(text) {
+function getParsedComment(text: string): string {
     const parser = new ExpensiMark();
-    return text.length <= CONST.MAX_MARKUP_LENGTH ? parser.replace(text) : _.escape(text);
+    return text.length <= CONST.MAX_MARKUP_LENGTH ? parser.replace(text) : lodashEscape(text);
```

Comment:
Please confirm if `underscore` and `lodash` implementation of `escape` produces the same result.

Reply:
Confirmed its the same 
Lodash 
`_.escape('fred, barney, & pebbles');
// => 'fred, barney, &amp; pebbles'`

Underscore
`_.escape('Curly, Larry & Moe');
=> "Curly, Larry &amp; Moe"`